changelog/8421.feature.rst

```diff
@@ -0,0 +1 @@
+``pytest.approx`` now works on Decimal within mapping and sequences.
```

Comment:
```suggestion
:func:`pytest.approx` now works on :type:`~decimal.Decimal` within mappings/dicts and sequences/lists.
```

Reply:
It looks like `:type:` is not well handled by Sphinx (?): https://github.com/pytest-dev/pytest/runs/2093692237?check_suite_focus=true

I'm not really knowledgeable with your set-up so I'm a bit lost :o The following changelog seems to pass CI:
```
:func:`pytest.approx` now works on `~decimal.Decimal` within mappings/dicts and sequences/lists.
```

Would you be OK with it?

Reply:
Gah, it probably needs to be `:class:`, sorry about that.

Reply:
`:class:` seems to be working locally, doing that now :+1: